src/Register.jsx

```diff
@@ -0,0 +1,46 @@
+import { Link } from "react-router-dom";
+import { tv } from "tailwind-variants";
+import { InputForm } from "./components/InputForm";
+import { Button } from "./components/Button";
+
+const RegisterPage = tv({
+  slots: {
+    base: "flex flex-col h-screen items-center pt-20",
+    headerText: "text-3xl font-bold mb-10",
+  },
+});
+
+const { base, headerText } = RegisterPage();
+
+export default function Login() {
+  return (
+    <main className={base()}>
+      <h1 className={headerText()}>ลงทะเบียน</h1>
+      <section className="w-4/5">
+        <form className="flex flex-col">
+          <InputForm
+            label="หมายเลขโทรศัพท์/อีเมล์"
+            id="username"
+            placeholder="name@email.com"
+          />
+          <InputForm label="รหัสผ่าน" type="password" id="password" hint="รหัสผ่านต้องมีความยาวอย่างน้อย 8 อักษร"/>
+          <InputForm label="ยืนยันรหัสผ่าน" type="confirmpassword" id="confirmpassword" />
```

Comment:
This type of InputFrom should be 'password'.
```suggestion
          <InputForm label="ยืนยันรหัสผ่าน" type="password" id="confirmpassword" />
```